src/components/EmptyState/EmptyState.tsx

```diff
@@ -17,7 +17,7 @@ export interface EmptyStateProps {
   image: string;
   /** The image to use for large screens */
   largeImage?: string;
-  /** The image to use for large screens */
+  /** The image width is contained within its wrapper */
```

Comment:
Would using the active voice like so work as well?

```suggestion
  /** Contain the image within its wrapper */
```